### DISCUSSION TITLE
Next Release?

### DISCUSSION THREAD

**Discussion by cameronpickham** [diesel-rs/diesel#3061] [Q&A] [answered]
Apologies if this is the wrong place to ask, it felt better than opening an issue. I am very interested in [this feature](https://github.com/diesel-rs/diesel/pull/2254), so I am wondering when the next release (that will hopefully include this) will be pushed out. Thanks!

**Comment by weiznich** [comment, marked as answer]
As this is a free time project we do not give any ETA's other than "when it's done" on releases.
